schema/ndt.json

```diff
@@ -29,6 +29,17 @@
           { "name": "server_kernel_version", "type": "STRING"},
           { "name": "tls", "type": "BOOLEAN"},
           { "name": "websockets", "type": "BOOLEAN"},
+          {
+            "fields": [
+              { "name": "as_numbers", "type": "STRING", "mode": "REPEATED", "description": "list of associated AS numbers"},
```

Comment:
Let's not optimize anything for the MOAS case.

One ASN is enough, and let's call it `asn`, because "as" is a SQL keyword.  If you *really* want to support the MOAS case, add a boolean indicating that it was part of a MOAS. But, honestly, even that may be overkill.